lib/libzfs/libzfs_sendrecv.c

```diff
@@ -2366,9 +2366,8 @@ zfs_send_cb_impl(zfs_handle_t *zhp, const char *fromsnap, const char *tosnap,
 		 * there was some error, because it might not be totally
 		 * failed.
 		 */
-		err = send_conclusion_record(outfd, NULL);
```

Comment:
Let's declare the variable here to keep the scope limited.

Reply:
Updated